src/sql/ResourceSelector.test.tsx

```diff
@@ -24,6 +24,10 @@ describe('ResourceSelector', () => {
     render(<ResourceSelector {...props} default="foo" value={defaultKey} fetch={fetch} onChange={onChange} />);
     expect(screen.queryByText('default (foo)')).toBeInTheDocument();
 
+    // TODO ResourceSelectorProps should be updated to make `label` required
```

Comment:
Is this todo for in this pr or a reminder for a follow up issue?

Reply:
Reminder for a follow up, could potentially be done in this PR though, just have to go through the places where this component is used and make sure it can be changed to a required prop.

Reply:
Actually, going to leave the prop as optional to match the InlineField component's label prop. I'll remove this TODO comment.